### PR TITLE
Crossbuild

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,11 +11,11 @@ PHONY = linux core akaros
 all: linux
 
 core:
-	cc $(CFLAGS) -falign-functions=4096 -falign-loops=8 -c ftqcore.c -o ftqcore.o
```
Comment (brho) [inline Makefile:13]: ha, we weren't even using $(CC) before!

**Review by brho — APPROVED** [review]

**Review by mjsottile — APPROVED** [review]
Looks good to me.  Has cycles.h been kept up to date with the FFTW version it was borrowed from many years ago?

**Comment by rminnich** [comment]
no it is not kept up, we need to get a newer one.

**Comment by brho** [comment]
regarding cycles.h, i had a small change a few years back to put in an lfence before the rdtsc.  see commit https://github.com/rminnich/ftq/commit/6881ce9e9d7a93002ffc5ba3cdc91ab79970507e for details.

**Pull request merged**